data/json/items/armor/arms_armor.json

```diff
@@ -279,5 +279,16 @@
     "material_thickness": 3,
     "environmental_protection": 1,
     "flags": [ "STURDY", "BELTED", "WATER_FRIENDLY" ]
+  },
+  {
+    "id": "armguard_cut_resistant",
+    "type": "ARMOR",
+    "name": "pair of cut-resistant arm sleeves",
```

Comment:
All new names need to be be updated with `"name": { "str": "pair of cut-resistant arm sleeves", "str_pl": "pairs of cut-resistant arm sleeves" }` syntax.